src/main.rs

```diff
@@ -807,13 +822,7 @@ fn compile_reverse(
                     file_path.set_extension(language);
                 }
 
-                if track_code_files.contains(&file_path) {
-                    return Err(format!(
-                        "Multiple locations point to code file {}",
-                        file_path.display()
-                    )
-                    .into());
-                } else {
+                if !track_code_files.contains(&file_path) {
```

Comment:
The `contains` call seems redundant as `insert` returns if the value was already present.

Reply:
Indeed... forgot that when removing the check/error that is not necessary in reverse mode